guides/integrate-nextauth.md

```diff
@@ -0,0 +1,121 @@
+	</iframe>
+</div>
+
+From the Admin Console:
+  1. Click the Realm selector on the top-left.
+  2. Click **Create New Realm**.
+  3. Type the name of your new realm. Click **Create Realm**.
+  4. Click **Switch to Realm**.
+  5. From the Realm's Home page, click Edit.
+
+### Create an Application
+
+Next, we we'll create a new [Application](https://developer.beyondidentity.com/docs/v1/platform-overview/architecture#applications-and-authenticator-configs) that contains the configuration for your end users:
+
+<div style={{position: 'relative', paddingBottom: 'calc(73% + 20px)', height: '0'}}>
+	<iframe src='https://demo.arcade.software/KmtiNsx4Z31MkogQdwST?embed&forceNoOpeningAnimation=true' frameBorder="0" style={{position: 'absolute', top: '0', left: '0', width: '100%', height: '100%'}}>
+	</iframe>
+</div>
+
+From the Admin Console: 
+  1. From the navigation bar, click **Applications**, then click **Add app**.
+  1. Type a name for your new Application.
+  1. Scroll to **Redirect URIs**, type `http://localhost:3000`.
+  1. Ensure **"Token Endpoint Auth Method"** is set to "Client Secret Post".
+  1. Ensure **"Subject"** is set to "id".
+  1. Verify that **"PKCE"** is disabled.
+  1. Click on the Authenticator Config tab, change the Configuration Type to `Hosted Web`.
+  1. Click Submit to save your changes.
+
+We will overwrite the *Redirect URI* value later in this process, so you can use the dummy value for now.
+
+At this point, your Beyond Identity Admin Console should be configured with a realm and an application set up.
+
+**Application**
+
+![application-config](/assets/nextauth-application-config.png)
+
+**Authenticator**
+
+![authenticator-config](/assets/nextauth-authenticator-config.png)
+
+### Create an Identity in the BI console
+
+We will create identities that can login to the NextAuth site here, and enroll them with a BI credential bound to their device.
+
+1. In the BI Admin console, click **"Identities"**
+1. If the user you wish to use already exists, skip to step 5.
+1. Click **"Add identity"**
+1. Input the desired Name, Username, and Email (All three are required)
+1. Next, we will send the user an enrollment email to bind a credential to their device.
+1. Follow the steps at [Send Enrollment Emails](send-enrollment) for each new identity. In the near future, this will become a push-button operation, but for now it involves sending commands via CURL.
+1. Each new identity will receive an Enrollment email, which they click on to bind a credential to their device (desktop, laptop, mobile, etc).
+
+## Configure NextAuth for OIDC
+
+### Install the free Plugin
+
+This guide is based on the popular [Available OAuth providers](https://authjs.dev/reference/providers/oauth-builtin) which is regularly updated and has many thousands of Active installations.
+
+Follow the installation steps specified in the [Beyond Identity](https://authjs.dev/reference/oauth-providers/auth0) Guide
```

Comment:
This link points to auth0 but we want to point to Beyond Identity

Reply:
Good catch! Thanks! I had to copy one of the other oauth provider links since beyond identity is still in review...